iohk-monitoring/src/Cardano/BM/Data/Configuration.lhs

```diff
@@ -42,6 +43,7 @@ import           Cardano.BM.Data.Rotation
 \subsubsection{Representation}\label{code:Representation}\index{Representation}\label{code:Port}\index{Port}
 \begin{code}
 type Port = Int
+type HostPort = (String, Int)
```

Comment:
why not HostPort = (String, Port) ?

Reply:
or, do we still need "Port"

Reply:
We still need `Port`, because it's used by other configuration